packages/cozy-client/src/CozyClient.js

```diff
@@ -991,6 +991,22 @@ class CozyClient {
     }
   }
 
+  /**
+   * Returns whether the client has been revoked on the server
+   */
+  async checkForRevocation() {
+    try {
+      await this.stackClient.fetchInformation()
+      return false
+    } catch (err) {
+      if (err.message && err.message.indexOf('Client not found') > -1) {
+        return true
+      } else {
+        return false
+      }
+    }
+  }
```

Comment:
What is the difference between this one and the one already in stackClient ? https://github.com/cozy/cozy-client/blob/fa0c1d6bea0c9e6d0f5a8c7cf0d74102982a88ab/packages/cozy-stack-client/src/CozyStackClient.js#L105 

Can we at least use this const https://github.com/cozy/cozy-client/blob/f67e6831e871dc0cc2d596c5a8387db1323cbf11/packages/cozy-stack-client/src/errors.js#L2 ?

Reply:
The one in CozyStackClient was a private helper used when running into a fetch error. It was not meant to used from the exterior.

Since the const needed to be use, I figured that it was better to have the method directly on CozyStackClient. So I removed the helper checkForRevocation and changed it into the checkForRevocation that I had put in CozyClient at first.

Reply:
👍